src/map/mob.c

```diff
@@ -4208,6 +4208,8 @@ static void mob_read_db_viewdata_sub(struct mob_db *entry, struct config_setting
 		entry->vd.head_bottom = libconfig->setting_get_int(it);
 	if ((it = libconfig->setting_get_member(t, "HairStyleId")) != NULL)
 		entry->vd.hair_style = libconfig->setting_get_int(it);
+	if (entry->vd.hair_style == 0)
```

Comment:
So What if I explicitly want hair style to be 0? It will still change to 1

Reply:
Don't know if the client supports hair style 0. But I'll change it to only default to 1 if `HairStyleId` was not defined.

Reply:
other games too use hercules, so it's better if we allow value